docs/stdlib/sequence.rst

```diff
@@ -89,32 +89,34 @@ Sequences
 
 .. note::
 
-   The sequence to be operated on by the functions above is specified
-   by a ``schema::ScalarType`` object.  If the sequence argument is
-   known ahead of time and does not change, the recommended way to pass
-   it is to use the :eql:op:`introspect` operator:
+    Any sequence that can be called upon by either :eql:func:`sequence_next`
+    or :eql:func:`sequence_rest` functions are specified by a
+    ``schema::ScalarType`` object. If the sequence argument is known ahead of
+    time and does not change, the recommended way to pass it is to use the
+    :eql:op:`introspect` operator:
 
-   .. code-block:: edgeql
+    .. code-block:: edgeql
 
-      select sequence_next(introspect MySequenceType);
-      # or
-      select sequence_next(introspect typeof MyObj.seq_prop);
+        select sequence_next(introspect MySequenceType);
+        # or
+        select sequence_next(introspect typeof MyObj.seq_prop);
```

Comment:
Changing all `sequence_next` references to `sequence_reset` to reflect the subject of this section. This content should all be replicated in the `sequence_next` documentation showing that function instead.

```suggestion
        select sequence_reset(introspect MySequenceType);
        # or
        select sequence_reset(introspect typeof MyObj.seq_prop);
```